docs/User/Frequently_asked_questions.md

```diff
@@ -295,6 +309,20 @@ One can also use “GET TASKRUNNER PEER STATUS” to display the status for the
 -   For RUN_SWARM task type, the status summary reports SWOP node UID, Number of SL PEERs this SWOP has spawned, and list of all SL node information \(UID, Status, Description\). For all other types of tasks, the status summary reports SWOP node status \(UID, Status, Description\).
 -   If there are failed PEERs, using its node UID, one can identify the container name/id from ‘LIST NODES’ command. With container name/id, user can debug the error with docker logs command.
 
+### How to assign task to all available SWOP peers?
+Use `ASSIGN TASK <task-name> TO <taskrunner name> WITH ALL PEERS` to trigger a task to run on *all available* SWOP peers listening on the specified Taskrunner. The syntax is used to run a given task on all available nodes at that point of time, without manually counting on how many nodes are available across the consortium.
+For more information on triggering this task from SLM-UI, see _Executing a task_ section in _HPE Swarm Learning
+Installation and Configuration Guide_. For more information on triggering this task from CLI mode, see _SWCI commands
+related to Taskrunner_.
+
+### How to target a task on a failed (task) SWOP peer?
+Use `ASSIGN TASK <task-name> TO <taskrunner name> TARGET <SWOP-UID>` to trigger a task to run on a particular SWOP. User can use `LIST NODES` command to identify the SWOP-ID of the desired SWOP Target.
+This can be useful to take corrective action by re-running a task that had failed earlier on a particular node.
+
+For more information on triggering this task from SLM-UI, see _Executing a task_ section in _HPE Swarm Learning
+Installation and Configuration Guide_. For more information on triggering this task from CLI mode, see _SWCI commands
+related to Taskrunner_.
+
```

Comment:
This FAQ question needs to be updated..  Remove this "SLM-UI does not work if you are running Podman container runtime on the host." under this question

### What are the limitations of SLM-UI?